src/documentation/pages/components/forms/search-input/examples/_portal_sizes.vue

```diff
@@ -0,0 +1,139 @@
+</div>`,
+        vue: `<div class="chi-form__item">
+  <ChiSearchInput value="Sample Text" size="xs" />
+</div>
+<div class="chi-form__item">
+  <ChiSearchInput value="Sample Text" size="sm" />
+</div>
+<div class="chi-form__item">
+  <ChiSearchInput value="Sample Text" size="md" />
+</div>
+<div class="chi-form__item">
+  <ChiSearchInput value="Sample Text" size="lg" />
+</div>`,
+        htmlblueprint: `<div class="chi-form__item">
+  <div class="chi-input__wrapper -icon--right">
+    <input class="chi-input chi-search__input -xs" type="search" value="Sample text" aria-label="search input" />
+    <button class="chi-button -icon -flat -xs -bg--none" aria-label="Search">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -xs" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>
+<div class="chi-form__item">
+  <div class="chi-input__wrapper -icon--right">
+    <input class="chi-input chi-search__input -sm" type="search" value="Sample text" aria-label="search input" />
+    <button class="chi-button -icon -flat -sm -bg--none" aria-label="Search">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -sm" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>
+<div class="chi-form__item">
+  <div class="chi-input__wrapper -icon--right">
+    <input class="chi-input chi-search__input -md" type="search" value="Sample text" aria-label="search input" />
+    <button class="chi-button -icon -flat -md -bg--none" aria-label="Search">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -sm" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>
+<div class="chi-form__item">
+  <div class="chi-input__wrapper -icon--right">
+    <input class="chi-input chi-search__input -lg" type="search" value="Sample text" aria-label="search input" />
+    <button class="chi-button -icon -flat -lg -bg--none" aria-label="Search">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -sm" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>`,
+      },
+    };
+  },
+})
+export default class PortalSizes extends Vue {}
```

Comment:
i believe this should be named the other way around, SizesPortal, for consistency with the other components